### PR TITLE
Migrate helm leftovers to kustomize

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Add istio inbound port exclusion annotation for ports 2020 and 2021 
- Add networkpolicy to kustomize files
- Add busola extensions to kustomize files
- Add otel-collector and fluent-bit dashboards to kustomize files
- Extend `manager_test` to check the existence of the new resources

**Related issue(s)**
https://github.com/kyma-project/telemetry-manager/issues/150

### CODE REVIEW

File: config/networking/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: telemetry-operator-pprof-deny-ingress
```
Comment (shorim) [inline config/networking/networkpolicy.yaml:4]: @hisarbalik the `telemetry-` prefix will be added by Kustomize https://github.com/kyma-project/telemetry-manager/blob/main/config/default/kustomization.yaml#L9. So, the name here should be only `operator-pprof-deny-ingress`

File: internal/resources/fluentbit/resources_test.go [hunk 2]
```diff
@@ -48,7 +53,7 @@ func TestMakeDaemonSet(t *testing.T) {
 	require.NotEmpty(t, daemonSet.Spec.Template.Spec.Containers[0].EnvFrom)
 	require.NotNil(t, daemonSet.Spec.Template.Spec.Containers[0].LivenessProbe, "liveness probe must be defined")
 	require.NotNil(t, daemonSet.Spec.Template.Spec.Containers[0].ReadinessProbe, "readiness probe must be defined")
-
+	require.Equal(t, daemonSet.Spec.Template.ObjectMeta.Annotations, expectedAnnotations, "annotations should contains istio port exclusion of 2020 and 2021")
```
Comment (chrkl) [inline internal/resources/fluentbit/resources_test.go:56]:
```suggestion
	require.Equal(t, daemonSet.Spec.Template.ObjectMeta.Annotations, expectedAnnotations, "annotations should contain istio port exclusion of 2020 and 2021")
```

**Pull request merged**